backend/companies/migrations/0001_initial.py

```diff
@@ -1,8 +1,9 @@
 # Generated by Django 3.2 on 2021-05-04 14:36
 
-from django.db import migrations, models
 import uuid
 
+from django.db import migrations, models
+
```

Comment:
This looks like a style change that could live in its own commit.

Reply:
Yea, that slipped in there :D I'll merge this to the original commit.